packages/vscode-extension/README.md

```diff
@@ -51,15 +50,18 @@ Verify you have the right prerequisites for building Teams apps and install some
 
 ### Create your project
 
-Use the Teams Toolkit in Visual Studio Code to set up your first app project. Create your app project using the following steps:
+Use the Teams Toolkit for Visual Studio Code to set up your first app project. Create your tab app project using the following steps:
 
-- Ensure you've installed the Microsoft Teams Toolkit
-- In Visual Studio Code, in `DEVELOPMENT` section click `Create a new Teams app`. Or open the Command Palette (Ctrl+Shift+P / ⌘⇧-P or View -> Command - Palette) and type "teams" and choose `Teams: Create New Project`.
-- Next, Add capabilities comes up. Select capability you want then Next.
-- Choose options according to your purpose, i.e., front-end hosting type, language, cloud resources and so on.
+- Ensure you've installed the Microsoft Teams Toolkit for Visual Studio Code
+- Select the Teams Toolkit icon in the Visual Studio Code sidebar.
+- Select `Create a new Teams app` button and at next prompt to also select `Create a new Teams app`.
+- Ensure that Tab is selected as the capability that you want to build in your app. Select OK.
```

Comment:
It should be replaced by something like "select the template based on your scenarios or required Teams capabilities", it isn't just for tab

Reply:
I added in the front that creating your first app becomes creating your first tab app, this way it is the quickest way for the 'first' app. so it is now tailored for tab. try to make the step one the simplest possible to get something run. and then then have all links to official doc. what do you think?